sopel/coretasks.py

```diff
@@ -1,9 +1,17 @@
 # coding=utf-8
-"""Tasks that allow the bot to run, but aren't user-facing functionality
+"""Core Sopel plugin that listen to basic IRC messages.
+
+This plugin allows the bot to run without user-facing functionality:
+
+* it handles client capability negotiation,
+* it handles client auth (both nick auth and server auth),
```

Comment:
```suggestion
* it handles client capability negotiation
* it handles client auth (both nick auth and server auth)
```

I've never been a fan of this style of list, and the remaining items didn't really carry through the pattern anyway. x)